lib/templates.ts

```diff
@@ -265,11 +265,13 @@ export async function createTemplate(command: CreateTemplateCommand): Promise<Fo
 
 /**
  * Get all form templates. Depending on the user permissions the function will return either all or a subset of templates.
+ * @param userID deprecated, retrieve from ability instead
  * @returns An array of Form Records
  */
 export async function getAllTemplates(
   ability: UserAbility,
-  userID: string
+  userID: string,
+  requestedWhere?: Prisma.TemplateWhereInput
```

Comment:
It looks like the calls to this function pass in the current user from ability. Though the current function signature is confusing. I updated it to use the userID param if it's passed. 
The calls to this function could be updated to remove userID but I figured it would be better to change something like that outside of this PR. Also maybe passing the userID could be useful, not sure?

Reply:
You're right, I misread the diff that the userID arg was removed from the function definition.
Since this function in only called from one other place in the code base we can safely go ahead and remove the userID arg and update the other calling module.